infra/nnfw/config/gbs.conf

```diff
@@ -18,5 +18,5 @@ url = http://download.tizen.org/snapshots/tizen/unified/latest/repos/standard/pa
 url = http://download.tizen.org/snapshots/tizen/base/latest/repos/standard/packages/
 
 [repo.tizen_one]
-url = http://nnfw.mooo.com/archive/tizen/
+url = http://13.125.34.93/archive/tizen/
```

Comment:
If it is built in AWS's builders, the cost of AWS's network may increase more than when the domain is used.

Reply:
@ragmani This config file is not for CI system only, for general runtime developer. And nnfw.mooo.com (will be deprecated) is forwarding to public IP.

Reply:
> And nnfw.mooo.com (will be deprecated)

@hseok-oh `mooo.com` service is no longer supported ? or just will be not used on our system ?

Reply:
We can't use `nnfw.mooo.com` any more. (please refer internal issue)